src/object/minishSizedArchway.c

```diff
@@ -0,0 +1,18 @@
+#include "global.h"
+#include "entity.h"
+#include "sprite.h"
+
+extern u32 CheckIsDungeon();
```

Comment:
this should probably be
```suggestion
extern u32 CheckIsDungeon(void);
```
and go in some header (say area.h? or just functions.h)